Jenkinsfile

```diff
@@ -1,115 +1,9 @@
-        // SAL user home
-        SAL_USERS_HOME = "/home/saluser"
-        // SAL setup file
-        SAL_SETUP_FILE = "/home/saluser/.setup_dev.sh"
-        // LTD credentials
-        user_ci = credentials('lsst-io')
-        LTD_USERNAME="${user_ci_USR}"
-        LTD_PASSWORD="${user_ci_PSW}"
-    }
-
-    stages {
-        stage ('Install dependencies') {
-            steps {
-                withEnv(["WHOME=${env.WORKSPACE}"]) {
-                    sh """
-                        source ${env.SAL_SETUP_FILE}
-
-                        pip install -r requirements.txt
-                    """
-                }
-            }
-        }
-
-        stage("Deploy documentation") {
-            when {
-                anyOf {
-                    branch "main"
-                    branch "develop"
-                }
-            }
-            steps {
-                script {
-                sh """
-                    source ${env.SAL_SETUP_FILE}
-                    # Create docs
-                    cd ./docsrc
-                    pip install -r requirements.txt
-                    sh ./create_docs.sh
-                    cd ..
-                    # Upload docs
-                    pip install ltd-conveyor
-                    ltd upload --product love-producer --git-ref ${GIT_BRANCH} --dir ./docs
-                """
-                }
-            }
-        }
-
-        stage ('Unit Tests and Coverage Analysis') {
-            steps {
-                // Pytest needs to export the junit report.
-                withEnv(["WHOME=${env.WORKSPACE}"]) {
-                    sh """
-                        source ${env.SAL_SETUP_FILE}
-
-                        # Install in development mode
-                        pip install -e .
-
-                        export OSPL_URI=\$(python -c "from lsst.ts import ddsconfig; print( (ddsconfig.get_config_dir() / 'ospl-std.xml').as_uri())")
-                        
-                        pytest --cov-report html --cov="${env.MODULE_NAME}" --junitxml=${env.XML_REPORT}
-                    """
-                }
-            }
-        }
-    }
-
-    post {
-        always {
-            // The path of xml needed by JUnit is relative to
-            // the workspace.
-            junit "${env.XML_REPORT}"
-
-            // Publish the HTML report
-            publishHTML (target: [
-                allowMissing: false,
-                alwaysLinkToLastBuild: false,
-                keepAll: true,
-                reportDir: 'htmlcov',
-                reportFiles: 'index.html',
-                reportName: "Coverage Report"
-            ])
-        }
-
-        cleanup {
-            // clean up the workspace
-            deleteDir()
-        }
-    }
-}
+@Library('JenkinsShared')_
+DevelopPipeline(
+    name: "love-producer",
+    module_name: "love.producer",
+    idl_names: [],
+    build_all_idl: true,
+    has_doc_site: false,
+    extra_packages: []
```

Comment:
You may omit lines 5 (`idl_names`) and 8 (`extra_packages`) since they contain the default values. See
https://github.com/lsst-ts/ts_jenkins_shared_library/blob/develop/vars/DevelopPipeline.groovy#L5-L15
Also, line 6 (`build_all_idl`) is ignored because of line 5. See
https://github.com/lsst-ts/ts_jenkins_shared_library/blob/develop/vars/DevelopPipeline.groovy#L30-L35
and
https://github.com/lsst-ts/ts_jenkins_shared_library/blob/develop/vars/DevelopPipeline.groovy#L116-L118
Please remove all three for clarity.